src/Analyzers/MSTest.Analyzers/Resources.resx

```diff
@@ -573,4 +573,10 @@ The type declaring these methods should also respect the following rules:
   <data name="UseConditionBaseWithTestClassMessageFormat" xml:space="preserve">
     <value>The attribute '{0}' which derives from 'ConditionBaseAttribute' should be used only on classes marked with `TestClassAttribute`</value>
   </data>
+  <data name="DuplicateDataRowTitle" xml:space="preserve">
+    <value>A duplicate 'DataRow' attribute is found</value>
```

Comment:
Maybe something like `Avoid duplicated 'DataRow' entries`? or `Do not duplicate 'DataRow' entries`